ui/ducks/send/swap-and-send-utils.ts

```diff
@@ -137,6 +141,10 @@ const QUOTE_VALIDATORS = [
 ];
 const SWAPS_API_VERSION = 'v2';
 
+const BASE_URL = process.env.SWAPS_USE_DEV_APIS
```

Comment:
is this going to be added to the extension too or just a temp dev override?

Reply:
It's already used in other places.